### PR TITLE
Fix: faq page and organised project structure 

### PR DESCRIPTION
**Pull request by Manoj0Marmat**
Issue #52 
Screen Recording 2022-10-07 at 11 31 32 PM

Solved 
faq page 
organised css, js files

### CODE REVIEW

**Comment by Manoj0Marmat** [comment]
@vanshgupta-2024 kindly review this

**Comment by Manoj0Marmat** [comment]
@robogyan-adgitm kindly review this

File: about.html [hunk 1]
```diff
@@ -18,15 +18,13 @@ <h1>Robogyan</h1>
         <div class="navbar">
           <ul class="navbar-links">
             <li>
-              <a href="index.html">Home</a>
+              <a href="./index.html">Home</a>
```
Comment (robogyan-adgitm) [inline about.html:21]: Generally asking, what does this do?

**Review by robogyan-adgitm — APPROVED** [review]

File: about.html (hunk 1 above)
Comment (Manoj0Marmat) [reply about.html:21]: this is best way to write this called it as relative path

Comment (Manoj0Marmat) [reply about.html:21]: @robogyan-adgitm there is not big difference i think we can also use index.html i always do this use ./index.html

**Comment by robogyan-adgitm** [comment]
Do not forget to star the repo @Manoj0Marmat

**Comment by Manoj0Marmat** [comment]
Sure

**Pull request merged**